app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -20,9 +22,6 @@ module WorksControllerBehavior
       self.show_presenter = Hyku::WorkShowPresenter
       self.work_form_service = Hyrax::WorkFormService
       self.search_builder_class = WorkSearchBuilder
-      self.iiif_manifest_builder = (
```

Comment:
Reading notes:

This is now covered in https://github.com/samvera-labs/hyrax-iiif_av/blob/main/app/controllers/concerns/hyrax/iiif_av/controller_behavior.rb#L61-L67